### ISSUE TITLE
Please help me

### ISSUE THREAD

**Issue by StarZigZag** [elebumm/RedditVideoMakerBot#204]
![image](https://user-images.githubusercontent.com/68636625/171872598-8d462f61-61e7-4871-9fcc-a5c84eef635d.png)
this error pops up everytime i try "pip3 install -r requirements.txt"

**Comment by Diwas37** [comment]
updating pip might help, copy and paste in terminal : 
python -m pip install --upgrade pip

**Comment by StarZigZag** [comment]
i just did, still the same error but without the upgrade pip part

**Comment by Diwas37** [comment]
https://github.com/elebumm/RedditVideoMakerBot/issues/24 
this might help

**Comment by jacesleeman** [comment]
Dumb question but do you have Python installed

**Comment by StarZigZag** [comment]
yeah i saw in the tread that they were gonna try with python 3.8 and they never said if it worked or not so i just asumed it did and tried

**Comment by jacesleeman** [comment]
> yeah i saw in the tread that they were gonna try with python 3.8 and they never said if it worked or not so i just asumed it did and tried

try py -3 -mpip RESTOFCOMMAND

**Comment by FreebieII** [comment]
> ![image](https://user-images.githubusercontent.com/68636625/171877363-669afee3-9df4-4061-89dd-a57992b8b282.png) now it does this instead

Try `pip install -r requirements.txt` , instead of `pip3`

**Comment by LukaHietala** [comment]
> > ![image](https://user-images.githubusercontent.com/68636625/171877363-669afee3-9df4-4061-89dd-a57992b8b282.png) now it does this instead
> 
> Try `pip install -r requirements.txt` , instead of `pip3`

If you still have problems with `pip` try to reinstall Python and while setting it up make sure that you added it to the Windows PATH.
![image](https://user-images.githubusercontent.com/95122845/172068724-0fb0edf4-5968-4790-9a05-ae27f200f764.png)

Click Add Python to the PATH.
